src/widget/loginscreen.cpp

```diff
@@ -39,6 +44,7 @@ LoginScreen::LoginScreen(QWidget *parent) :
     // permanently disables maximize button https://github.com/tux3/qTox/issues/1973
     this->setWindowFlags(windowFlags() &! Qt::WindowMaximizeButtonHint);
     this->setFixedSize(this->size());
+    ui->newUsername->setValidator(new QRegExpValidator(QRegExp("[a-z]"), this));
```

Comment:
1. Use `QRegularExpression` instead of `QRegExp`.
2. Shouldn't the regex be case insensitive `QRegularExpression("[a-zA-Z]")`? What about decimal numbers?